money-rails.gemspec

```diff
@@ -26,8 +26,8 @@ Gem::Specification.new do |s|
 
   s.require_path = "lib"
 
-  s.add_dependency "money",         "~> 6.13.2"
-  s.add_dependency "monetize",      "~> 1.9.0"
+  s.add_dependency "money",         "< 7.0"
```

Comment:
@semmons99 the `<` constraint seems super loose (versus `~> major.minor` on the current version). would this gem really function with v1 of `money` and v0.x of `monetize`?

Reply:
good catch. we should probably just make it `~> 6.13` right?